source/manual/fall-back-to-mirror.html.md

```diff
@@ -1,70 +1,75 @@
 ---
 owner_slack: "#govuk-2ndline"
-title: Fall back to the static mirrors
+title: GOV.UK CDN static mirrors
 section: Deployment
 layout: manual_layout
 parent: "/manual.html"
 ---
 
-We maintain a static copy of most of the site, which gets used by the content delivery
-network (CDN) whenever origin (the application server) times out or serves an error
-response.
+GOV.UK uses the Fastly content delivery network (CDN) to handle most requests to GOV.UK. When a user’s request first reaches the Fastly CDN, Fastly will check to see if it has a response for the request in its own cache.
 
-This process is handled by our CDN config and is entirely transparent to us and
-our users. It happens multiple times a day, for lots of different reasons.
+If this is the first time the page has been requested, Fastly will:
 
-This is why we refer to switching off Nginx on the origin cache machines as
-"falling back to the mirrors".
+- forward the request to the GOV.UK servers, otherwise known as Origin
+- cache the response locally
+- return the response to the user
 
-## Hosting
+Subsequent requests for the same resource will never hit the origin server, as long as Fastly keeps the response in its cache. Instead, Fastly will return what it has in its cache. This means that Fastly absorbs a lot of traffic and reduces the load on Origin.
 
```

Comment:
I think everything above here should be covered elsewhere (eg https://docs.publishing.service.gov.uk/manual/cdn.html). 

Otherwise there's a risk of duplication/diluting this page. It should only be about the mirror

Reply:
ok, will remove and add link to that page